variables.tf

```
@@ -40,6 +40,18 @@ variable "admin_user" {
   default     = "postgres"
 }
 
+variable "resource_proxy" {
+  type = object({
+    cpu     = number,
+    memory  = number
+  })
+  description = "Postgres proxy resources"
+  default = {
+    cpu         = 500
+    memory      = 1024
```

These are the actual defaults, so I suggest we leave them at that, then we can change it when in Minerva
```suggestion
    cpu         = 200
    memory      = 128
```

---

I think we should add `validation` as well.
Here is an example from Presto:
```hcl
variable "resource_proxy" {
  type = object({
    cpu    = number,
    memory = number
  })
  default = {
    cpu    = 200,
    memory = 128
  }
  description = "Presto proxy resources"
  validation {
    condition     = var.resource_proxy.cpu >= 200 && var.resource_proxy.memory >= 128
    error_message = "Proxy resource must be at least: cpu=200, memory=128."
  }
}
```